### PR TITLE
Update Dependencies & Added Versioning

### PR DESCRIPTION
**Pull request by s0kil**
Does Crystal handle versioning similar to NPM, where Minor and Patch versions are always the latest available?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ dist
 mint
 docs
 bin
+TODO.md
```
Comment (Sija) [inline .gitignore]: Why?

File: shard.yml [hunk 2]
```diff
@@ -11,13 +11,13 @@ dependencies:
     version: 0.2.1
   baked_file_system:
     github: schovi/baked_file_system
-    version: 0.9.7
+    version: 0.9.8
```
Comment (Sija) [inline shard.yml]: It might be worth considering usage of `~>` operator instead of fixed versions.

File: .gitignore (hunk 1 above)
Comment (s0kil) [reply .gitignore]: Done

File: .gitignore [hunk 3]
```diff
@@ -4,4 +4,4 @@ mint-stuff
 dist
 mint
 docs
-bin
+bin
```
Comment (Sija) [inline .gitignore]: That change should be reverted.

File: shard.yml (hunk 2 above)
Comment (gdotdesign) [reply shard.yml]: yeah probably we should do that :slightly_smiling_face:

**Comment by gdotdesign** [comment]
Thanks :+1: and thanks for the review (as always) @Sija :wink:

**Pull request merged**